sympy/functions/combinatorial/factorials.py

```diff
@@ -1106,8 +1106,10 @@ def _eval_rewrite_as_factorial(self, *args, **kwargs):
         denom = Mul(*(factorial(x) for x in args))
         return numer/denom
 
-    def _eval_rewrite_as_binomial(self, *args, **kwargs):
-        pass
+    def _eval_rewrite_as_binomial(self, *k, *j, **kwargs):
+        from sympy import binomial, Add, Mul
+        Mul(*[binomial(j,k) for j,k in zip([Add(*k[:i]) for i in range(1,len(k)+1)],k)])
```

Comment:
@Arpan612 dude if you're going to plagiarize my code atleast don't copy it with the errors 😞 ... @sylee957 you might wanna look into this ...